eip-2930.md

```diff
@@ -0,0 +1,132 @@
+---
+eip: 2930
+title: Optional access lists
+author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
+discussions-to: TBD
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-08-29
+requires: EIP 2718; EIP 2929
+---
+
+## Simple Summary
+
+Adds a transaction type which contains an access list, a list of addresses and storage keys that the transaction plans to access. Accesses outside the list are possible, but become more expensive. Intended as a mitigation to contract breakage risks introduced by EIP 2929 and simultaneously a stepping stone toward broader use of access lists in other contexts.
+
+## Abstract
+
+We introduce a new [EIP-2718](https://github.com/ethereum/EIPs/pull/2718) transaction type, with the format `rlp([3, [nonce, gasPrice, gasLimit, to, value, data, access_list, senderV, senderR, senderS]])`.
```

Comment:
There is currently a debate on whether internal EIP links should be `[EIP-2718](./eip-2718.md)` or `[EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)`.  What you have here isn't either so is definitely not in line with the current "standard".  😄